source/scripts/models/Hero.js

```diff
@@ -52,6 +54,10 @@ export default class Hero extends Pixi.Sprite {
         this.timeSinceAttack = this.attackCooldownTime
 
         this.monsterRank = "warrior"
+
+        this.collider = new CircleCollider(this.position, this.scale, this.rotation)
+        this.collider.parent = this
+        //this.addChild(this.collider)
```

Comment:
It probably does work! I was using these two lines of codes at separate times with different organizations in mind. I didn't even notice they were practically identical.